source/features/close-out-of-view-modals.tsx

```diff
@@ -13,7 +14,7 @@ function init(): void {
 	// so the selector is inverted
 	delegate('.details-overlay:not([open]) > summary', 'click', event => {
 		// What comes after <summary> is the dropdown
-		observer.observe(event.delegateTarget.nextElementSibling!);
+		observer.observe(select('.dropdown-menu', event.delegateTarget.parentElement!)!);
```

Comment:
I don’t like mixing `select` with other dom traversal properties on the same line, it makes it hard to follow. Also you can use `({delegateTarget})` as a parameter to shorten the line further
```suggestion
		observer.observe(delegateTarget.parentElement!.querySelector('dropdown-menu')!);
```